tcf_website/static/common/sortClasses.js

```diff
@@ -37,10 +38,16 @@ const sortByNumber = () => {
 const sortByRating = () => {
     activeSort = "rating";
     if (!$("#rating-sort-btn").hasClass("active")) {
-        $("#number-sort-btn").removeClass("active");
+        if ($("#number-sort-btn").length) {
```

Comment:
Same thing as before - I assume these are just proxies for checking whether or not the page that's using the sort function has these fields?

Reply:
Yeah, I added more comments to document this in case it was confusing.